keps/sig-network/563-dual-stack/README.md

```diff
@@ -1360,25 +1398,28 @@ This capability will move to stable when the following criteria have been met.
 
 * **Can the feature be disabled once it has been enabled (i.e. can we roll back
   the enablement)?**
-  Yes.
+  Yes. If we disable the feature gate, we must remove the CLI parameters. An
+  older client won't see or be able to use the new fields.
 
 * **What happens if we reenable the feature if it was previously rolled back?**
-  Similar to enable it the first time on a cluster.
+  Similar to enabling it the first time on a cluster. We don't load balance
+  across IP families, and with no selectors we don't get endpoints. If you use
+  the feature flag to turn off dual-stack, we do not edit your service. If you
+  disable dual-stack from the controller manager, the service will be given
+  single-stack endpoints.
 
 * **Are there any tests for feature enablement/disablement?**
   The feature is being tested using integration tests with gate on/off. The
   tests can be found here: https://github.com/kubernetes/kubernetes/tree/master/test/integration/dualstack
 
-  The feature is being tested on -some of - the cloud providers and kind.
-   1. https://testgrid.k8s.io/sig-network-dualstack-azure-e2e. This has all dualstack tests on azure.
+  The feature is being tested on a cloud provider and kind.
+   1. azure dual-stack e2e: https://testgrid.k8s.io/sig-network-dualstack-azure-e2e
    2. kind dual-stack iptables: https://testgrid.k8s.io/sig-network-kind#sig-network-kind,%20dual,%20master
    3. kind dual-stack ipvs: https://testgrid.k8s.io/sig-network-kind#sig-network-kind,%20ipvs,%20master
 
 ### Rollout, Upgrade and Rollback Planning
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
```

Comment:
What about workload failures? If I upgrade and suddently 10% of my workloads start crashlooping because they are stupid and don't know what to do with a AAAA record from DNS, I want to disable and/or rollback. Again this probably doesn't apply much in the upgrade case because there are no dual-stack services, but in the re-enablement case it does.

Reply:
if you change the IPFamilyPolicy to SingleStack you will be back to the initial situation